src/studio/src/designer/frontend/packages/schema-editor/src/components/InputField.tsx

```diff
@@ -79,10 +82,11 @@ export function InputField(props: IInputFieldProps) {
       <Grid item xs={4}>
         <FormControl>
           <Input
-            id={`${baseId}-key-${label}`}
-            value={label}
+            id={`${baseId}-key-${value}`}
```

Comment:
Will this be ideal? Should the Id change every time a value changes?

Reply:
Is there a better way to get unique ids? they are mostly used in unit tests

Reply:
If the key's are only used in unit-tests they should not be in the rendered HTML.
If you make a store that gives you a unique number every time it is called you can do something like this:
`const inputId = baseId + '-key-' + getUniqueNumber();`
Then you can use this in lables and inputs later in the render block.
An unique number store can be written as such:
```js
let unusedNumber = 0;
const getUniqueNumber = () {
  return unusedNumber++;
}
export default getUniqueNumber;
```

Reply:
For tests you can add a function that sets the unusedNumber back to 0 in the beforeEach block.

Reply:
But then id will change on each render? is that any better? I would think that will make unit testing harder

Reply:
You can also send the id from outside the component context as a prop.
It is OK that the id changes on each render ( as the component will re-render when the value changes anyway )
The main change is that it is higher up in the hierarchy.
There is no guard against instances where 2 values are equal.
The id and value of a field should not be linked together, this can confuse a <label for=""> attribute later.